tests/fv3_conf/control_run.IN

```diff
@@ -88,8 +88,10 @@ fi
 if [ $TILEDFIX = .true. ]; then
     if [ $CPLLND == .true. ]; then
         cp    @[INPUTDATA_ROOT]/FV3_fix_tiled/@[ATMRES]/@[ATMRES]*.nc INPUT/.
+        cp    /scratch2/NCEPDEV/stmp1/Sanath.Kumar/repo_viirs30s_bnuv2/sfc.@[ATMRES]/*color* INPUT/. 
     else
         cp    @[INPUTDATA_ROOT]/FV3_fix_tiled/@[ATMRES]/@[ATMRES]*.nc .
+        cp    /scratch2/NCEPDEV/stmp1/Sanath.Kumar/repo_viirs30s_bnuv2/sfc.@[ATMRES]/*color* . 
```

Comment:
Same here

Reply:
Once the soil color data is in that directory, I will remove those additional lines.

Reply:
This line needs to be removed when the soil color data are put in @[INPUTDATA_ROOT]/FV3_fix_tiled